src/internal/constants.ts

```diff
@@ -8,6 +8,11 @@ export const version = '1.10.0';
  */
 export const defaultSDKVersionForCompatCheck = '2.0.1';
 
+/**
+ * This is the SDK version when selectMedia API - VideoAndImage is supported on mobile.
+ */
```

Comment:
Shivangi - As part of this PR https://github.com/OfficeDev/microsoft-teams-library-js/pull/591 Sonal has done some changes to move out of sdk version. Now in your PR we are adding bigger version which doesn't align to model where we are going. Could you please sync with Sonal and update your PR without adding new version.